### ISSUE TITLE
Hibernate doesn't see props

### ISSUE THREAD

**Issue by LIttleAncientForestKami** [caroo7/Caro-robot#19]
INFO: HHH000206: hibernate.properties not found
sie 29, 2016 3:33:53 PM org.hibernate.cfg.Environment buildBytecodeProvider

**Comment by LazyScoundrel** [comment]
@caroo7 could you comment on this? I remember you've said smth about this to me

**Comment by caroo7** [comment]
Yes, right now we load hibernate properties from simple properties file, we put them to java.util.Properies and finally we use in entity manager factory bean.
As i read, hibernate always try to find hibernate.properties file during inititialization, so I think it's not anything important.

**Comment by LIttleAncientForestKami** [comment]
So... the resolution will be?
